docs/getting-started/get-logs-into-loki.md

```diff
@@ -0,0 +1,74 @@
+# Get logs into Loki
+
+After you [install and run Loki](./installation/local.md), you probably want to get logs from other applications into it.
+
+To get application logs into Loki, you need to edit the [Promtail](./clients/promtail/README.md) config file.
+
+Detailed information about configuring Promtail is available in [Promtail configuration](./clients/promtail/configuration.md).
+
+The following instructions should help you get started.
+
+1. If you haven't already, download a Promtail configuration file and put it in the same directory as promtail.exe.
+
+```
+wget NEED DOWNLOAD LINK
```

Comment:
I think you need the link here.

Reply:
I need you guys to add the link, because I could not find it. Them. There's more than one place where we need these.

Reply:
Depending on the system running promtail, it'll be one of these at a url like
`https://github.com/grafana/loki/releases/download/v1.5.0/promtail-linux-amd64.zip`

![image](https://user-images.githubusercontent.com/8173478/85179146-31262c80-b24e-11ea-8012-01d0f3ad1d0c.png)

Reply:
That doesn't look like the download link for a config file.

Reply:
Uh, good point. Idk what I was thinking.
I think you can reuse
```
wget https://github.com/grafana/loki/blob/master/cmd/promtail/promtail-local-config.yaml
```